releasing/cloudbuild.sh

```diff
@@ -1,131 +1,32 @@
 #!/bin/bash
 #
-# Usage (from top of repo):
+# This script is called by Kustomize's Cloud Build release pipeline.
+# It installs jq (required for release note construction)
+# and then runs goreleaser (http://goreleaser.com).
 #
-#  releasing/cloudbuild.sh TAG [--snapshot]
+# To test it locally, run it in a goreleaser container:
 #
-# Where TAG is in the form
+#   # Get goreleaser image from cloudbuild.yaml
+#   export GORELEASER_IMAGE=goreleaser/goreleaser:v0.179.0
 #
-#   api/v1.2.3
-#   kustomize/v1.2.3
-#   cmd/config/v1.2.3
-#   ... etc.
+#   # Drop into a shell
+#   docker run -it --entrypoint=/bin/bash  -v $(pwd):/go/src/github.com/kubernetes-sigs/kustomize -w /go/src/github.com/kubernetes-sigs/kustomize $GORELEASER_IMAGE
 #
-# Cloud build should be configured to trigger on tags
-# matching:
+#   # Run this script in the container, where $TAG is the tag to "release" (e.g. kyaml/v0.13.4)
+#   ./releasing/cloudbuild.sh $TAG --snapshot
```

Comment:
Question, I did this with `./releasing/cloudbuild.sh kyaml/v0.13.4 --snapshot`, and the output has a bunch of things related to the kustomize module, e.g.

```
   • parsing tag      
         • DEPRECATED: 'kustomize/v4.5.3' is not SemVer-compatible and may cause other issues in the pipeline, check https://goreleaser.com/deprecations#skipping-semver-validations for more info
```

and 

```
      • github/gitlab/gitea releases
         • pre-release for tag kustomize/v4.5.3 set to false
```

is that expected?

Reply:
Surprising indeed, though not a new problem with the script. I discovered and added the [GORELEASER_CURRENT_TAG env var](https://goreleaser.com/customization/build/#define-build-tag) to fix this.